### ISSUE TITLE
Add baseline models for time series

### ISSUE THREAD

**Issue by Gerhardsa0** [Safe-DS/Library#518]
### Is your feature request related to a problem?

When working with time series, statistical models give a good baseline for prediction. So a user can compare his neural network with a statistical model easily.

### Desired solution

I am not sure where the baseline models should be defined and if they should have an own class.  They should be used like the existing Models in the Safe-DS Library.

**Comment by lars-reimann** [comment]
> if they should have an own class

No, the baseline models are instances of the classes you define. Instead, provide global functions to create the baseline model.
